docs/src/app/components/nav-side-link.tsx

```diff
@@ -16,8 +16,8 @@ interface CustomNavLinkProps {
 
 const getBgColor = (currentTheme: CurrentTheme) => {
   switch (currentTheme) {
-    case 'webappTheme': return 'blue.100'
-    case 'tottTheme': return 'blue.800'
+    case 'webappTheme': return 'background.button.ghost-hover'
+    case 'tottTheme': return 'background.button.ghost-hover'
     default: return 'bg.filled'
   }
 }
```

Comment:
You can probably remove this entire block of code and just set bgColorHover to background.button.ghost-hover directly